pmm-app/src/pmm-check/CheckPanel.tsx

```diff
@@ -113,14 +113,14 @@ export class CheckPanel extends PureComponent<CheckPanelProps, CheckPanelState>
           <>
             <div className={styles.header}>
               <div className={styles.title} data-qa="db-check-panel-title">
-                {title ?? ''}
+                {title}
               </div>
               <ButtonWithSpinner
                 onClick={this.handleRunChecksClick}
                 isLoading={isRunChecksRequestPending}
                 disabled={hasNoAccess}
                 className={styles.runChecksButton}
-                variant="secondary"
+                variant="primary"
```

Comment:
I think the variant `primary` is the default, so this line could be removed

Reply:
So, `explicit vs implicit` doesn't apply here? A go developer would argue, but I won't )